SteamReview/SteamReview/Steam.cpp

```diff
@@ -185,24 +205,24 @@ FGame FSteam::AddNewGame()
 	std::cout << "Please enter Day ";
 	int InDay;
 	std::cin >> InDay;
-	InDay = ValidateInput(InDay, 0, 31);
+	ValidateInput(InDay, 1, 31);
 
 	std::cout << "Please enter Month ";
 	int InMonth;
 	std::cin >> InMonth;
-	InMonth = ValidateInput(InMonth, 0, 12);
+	ValidateInput(InMonth, 1, 12);
 
 	std::cout << "Please enter Year ";
 	int InYear;
 	std::cin >> InYear;
-	InYear = ValidateInput(InYear, 1990, 2022);
+	ValidateInput(InYear, 1990, 2022);
 
 	//Creating a Game
-	return FGame(InGameName, InStudioName, InDay, InMonth, InYear);
+	InGame = FGame(InGameName, InStudioName, InDay, InMonth, InYear);
```

Comment:
This is weird, you're treating this parameter as an Out parameter, so maybe change its name to reflect what you're doing with it.

Reply:
Yup! I added the previous comment thinking it was indeed an In parameter, but it's not. So maybe it should be named OutGame. 

Maybe it would be good to rename the function as well.

`AddNewGame` implies that the game is being added somewhere, but it's not. Instead, you are creating a new game, so why not call this `CreateNewGame`?